DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/SnapshotTestCase.swift

```diff
@@ -35,6 +35,21 @@ internal class SnapshotTestCase: XCTestCase {
         return viewController
     }
 
+    func showNavigationController(for fixture: Fixture) -> UINavigationController? {
```

Comment:
/question This method and `AppDelegate.showNavigationController` are not used, should they be removed?

Reply:
Totally, I forgot to delete them. It's fixed now ✅